fw/tls.c

```diff
@@ -805,6 +806,10 @@ tfw_tls_conn_send(TfwConn *c, TfwMsg *msg)
 	 */
 	int ss_flags = READ_ONCE(msg->ss_flags);
 
+	tfw_connection_get(c);
```

Comment:
Why we need it here? Could you explain it please

Reply:
According to our private discussion currently all works fine in this place (no bugs), this is done to handle reference inside function.